docs/user/resources/04-100-10-aws-nfs-volume-restore.md

```diff
@@ -0,0 +1,51 @@
+# AwsNfsVolumeRestore Custom Resource
+
+The `awsnfsvolumerestore.cloud-resources.kyma-project.io` namespaced custom resource (CR) describes the AWS EFS file 
+system full restore operation on the same existing EFS file system.
+
+Out-of-place restore and Item-Level restore are not supported at this version of the `Cloud Manager`.
```

Comment:
```suggestion
Out-of-place restore and Item-Level restore are not supported by the `Cloud Manager`.
```
It's good practice to avoid hinting at planned features, so that we're never accidentally colliding with Product Standard [UA-030](https://wiki.one.int.sap/wiki/display/PSUA/UA-030).